variables.tf

```
@@ -101,6 +101,15 @@ variable "monitor_iam_access" {
   }
 }
 
+variable "sns_security_subscription" {
+  type = list(object({
+    sns_endpoint          = string
+    sns_endpoint_protocol = string
```

```suggestion
    endpoint = string
    protocol = string
```

No need to include an `sns` prefix if the variable is about SNS already.

Does it make sense to trim `endpoint_protocol` down to just `protocol` too?